### ISSUE TITLE
Nexus: RMG input test failing on Summit

### ISSUE THREAD

**Issue by mcbennet** [QMCPACK/qmcpack#2958]
**Describe the bug**
For latest develop branch, the nexus rmg_input test fails on Summit.
```
mcbennet@login1:QMCPACK(develop)$ ./nexus/bin/nxs-test -R rmg_input

 1/1 rmg_input...............................   Failed  1.28 sec
Test name     : rmg_input
Test sublabel : test_generate
Test exception: "AssertionError: "
Test backtrace:
  File "./nexus/bin/nxs-test", line 478, in run
    self.operation()
  File "./nexus/bin/nxs-test", line 1242, in rmg_input
    nunit('generate')
  File "./nexus/bin/nxs-test", line 349, in nunit
    run_external_unit_test(test_name,unit_test)
  File "./nexus/bin/nxs-test", line 388, in run_external_unit_test
    unit_test()
  File "/autofs/nccs-svm1_home1/mcbennet/Codes/src/QMCPACK/nexus/tests/unit/test_rmg_input.py", line 1267, in test_generate
    assert(check_object_eq(gr,gr_ref))


0% tests passed, 1 tests failed out of 1

Total test time = 1.28 sec
```
 

**To Reproduce**
1. Git hash: 05dfe95
2. N/A
3. ./nexus/bin/nxs-test -R rmg_input
4. None

**System:**
 - system name: summit
 - modules loaded
```
 Currently Loaded Modules:
  1) xl/16.1.1-5   2) spectrum-mpi/10.3.1.2-20200121   3) hsi/5.0.2.p5   4) xalt/1.2.1   5) lsf-tools/2.0   6) darshan-runtime/3.1.7   7) DefApps   8) python/3.6.6-anaconda3-5.3.0
```
 - other systems where this is reproducible: 

**Additional context**
Important note. The rmg_input test passes on my laptop. So the issue is somehow summit-specific or at least not general.

**Comment by jtkrogel** [comment]
What is the output when you do `nxs-test -R rmg_input --verbose`?

**Comment by mcbennet** [comment]
Negligible kpoint coordinates appear to be the issue here.  
[verbose.txt](https://github.com/QMCPACK/qmcpack/files/6044505/verbose.txt)

**Comment by prckent** [comment]
Can you confirm that the non-rmg tests are passing for you on summit with this configuration?

**Comment by jtkrogel** [comment]
I'm getting a 91% pass rate with a minimal python environment on Summit (numpy only).  Some of the issues (not Chandler's) appear to be unprotected module imports.

I will fix these issues, though likely not today.

**Comment by mcbennet** [comment]
Just for reference, I see 98% pass rate (where rmg_input is the only failure) when using the modules I listed above.

**Comment by prckent** [comment]
Closed by #2983 

Please make a new issue if any resurface
